core/src/shred_fetch_stage.rs

```diff
@@ -171,6 +171,7 @@ impl ShredFetchStage {
                     PACKET_COALESCE_DURATION,
                     true, // use_pinned_memory
                     None, // in_vote_only_mode
+                    false,
```

Comment:
Please follow the code style.

The comments makes the life of someone reading this code a lot easier and also makes bugs/mistakes less likely.

Reply:
see above.